docs/gathered-data.md

```diff
@@ -13,1039 +13,1984 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
-  - Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/pod](./insights-archive-sample/pod)
+- [docs/insights-archive-sample/events](./insights-archive-sample/events)
+
+### Location in archive
+| Version   | Path																    |
+| --------- | ---------------------------------------------------------------------- |
+| >= 4.3.0  | config/pod/{namespace}/{pod}.json										|
+| >= 4.8.2  | config/pod/{namespace}/logs/{pod}/{container}_{current|previous}.log   |
+| >= 4.8.2  | events/																|
+
+> **Note**
+> This Gatherer generates 3 types of reports.
+
+### Config ID
+`clusterconfig/operators_pods_and_events`
+
+### Released version
+- 4.3.0
+
+### Backported versions
+- 4.6.35+
+- 4.7.11+
+
+### Notes
+This used to be part of `ClusterOperators`, it was split generating this gather at `4.8.2`.
+It used to be Specs for `ClusterOperators`, and it was initially included at `4.3` and backported to `4.2.10`.
 
 
 ## ClusterOperators
 
-collects all the ClusterOperators definitions and their resources.
+Collects all the ClusterOperators definitions and their resources.
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
 
-* Location of operators related resources: config/clusteroperator/{group}/{kind}/{name}
-* Location of operators in archive: config/clusteroperator/
-* Location of operators related resources in older versions: config/clusteroperator/{kind}-{name}
-* See: docs/insights-archive-sample/config/clusteroperator
-* Id in config: clusterconfig/operators
-* Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### Sample data
+- [docs/insights-archive-sample/config/clusteroperator](./insights-archive-sample/config/clusteroperator)
+
+### Location in archive
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.3.0  | config/clusteroperator/ 									|
+| < 4.7.0   | config/clusteroperator/{kind}-{name}.json 					|
+| >= 4.7.0  | config/clusteroperator/{group}/{kind}/{name}.json 			|
+
+### Config ID
+`clusterconfig/`
+
+### Released version
+- 4.2.0
+
+### Backported versions
+None
+
+### Notes
+The `ClusterOperators` were used to also collect `pods` and `events`, it changed at the `4.8.2` release. The `pods`
+and `events` gathering were introduced in `4.3.0` and backported to `4.2.10`.
+
+* Spec config for CO resources was introduced at `4.7.0` and backported to`4.6.16+`
```

Comment:
This is becoming a little chaotic. How about structuring this as a changelog with one entry per change? Something like

```
### Changes

* The gathering of `pods` and `events` was moved to a separate gatherer, `ClusterOperatorPodsAndEvents`, in `4.8.2`.
* Spec config for CO resources was introduced at `4.7.0` and backported to `4.6.16+`
* The `pods`
and `events` gathering were introduced in `4.3.0` and backported to `4.2.10`.
```